introduction.md

```diff
@@ -1,6 +1,7 @@
 I am the father of dragons
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?